websocket-lite/examples/hello-world-client.rs

```diff
@@ -43,14 +43,9 @@ async fn run() -> Result<()> {
     Ok(())
 }
 
-fn main() {
-    let rt = tokio::runtime::Runtime::new().unwrap();
-
-    rt.spawn(async {
-        run().await.unwrap_or_else(|e| {
-            eprintln!("{}", e);
-        })
-    });
-
-    rt.shutdown_on_idle();
+#[tokio::main]
```

Comment:
Can you update this to use `spawn` again? The idea is to use `spawn` to ensure this stuff works across threads (per issue #1).

This is what I had:
```rust

fn main() {
    let mut rt = tokio::runtime::Runtime::new().unwrap();

    rt.block_on(rt.spawn(async {
        run().await.unwrap_or_else(|e| {
            eprintln!("{}", e);
        })
    }))
    .unwrap()
}
```